connectioncmd/input.go

```diff
@@ -0,0 +1,390 @@
+package connectioncmd
+
+import (
+	"fmt"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/hazelcast/hazelcast-commandline-client/config"
+	"os"
+	"strings"
+)
+
+var (
+	selectedItemStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("6"))
+	noStyle           = lipgloss.NewStyle()
+
+	focusedButton = selectedItemStyle.Copy().Render("[ Submit ]")
+	blurredButton = fmt.Sprintf("%s", noStyle.Render("[ Submit ]"))
+)
+
+type InputModel struct {
+	focusIndex     int
+	inputs         []textinput.Model
+	cursorMode     textinput.CursorMode
+	quitting       bool
+	state          int
+	config         *config.Config
+	connectionType string
+}
+
+func ViridianInput(config *config.Config) InputModel {
+	m := InputModel{
+		inputs:         make([]textinput.Model, 6),
+		config:         config,
+		connectionType: "Hazelcast Viridian",
+		state:          0,
+		cursorMode:     textinput.CursorStatic,
+	}
+	var t textinput.Model
+	for i := range m.inputs {
+		t = textinput.New()
+		t.CursorStyle = selectedItemStyle
+		switch i {
+		case 0:
+			t.Prompt = "- Cluster Name: "
+			t.Focus()
+			t.PromptStyle = selectedItemStyle
+			t.TextStyle = selectedItemStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 1:
+			t.Prompt = "- Discovery Token: "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 2:
+			t.Prompt = "- CA Certificate Path: "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 3:
+			t.Prompt = "- SSL Certificate Path: "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 4:
+			t.Prompt = "- SSL Key Path: "
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		case 5:
+			t.Prompt = "- SSL Password: "
+			t.EchoMode = textinput.EchoPassword
+			t.EchoCharacter = '•'
+			t.PromptStyle = noStyle
+			t.TextStyle = noStyle
+			t.SetCursorMode(textinput.CursorStatic)
+		}
+		m.inputs[i] = t
+	}
+	return m
+}
+
+func CloudInput(config *config.Config) InputModel {
+	m := InputModel{
+		inputs:         make([]textinput.Model, 3),
+		config:         config,
+		connectionType: "Hazelcast Cloud",
+		state:          0,
+		cursorMode:     textinput.CursorStatic,
+	}
+	var t textinput.Model
+	for i := range m.inputs {
+		t = textinput.New()
+		t.CursorStyle = selectedItemStyle
+		switch i {
+		case 0:
```

Comment:
Just a suggestion, creating constant for these values or providing a comment would help for readability. There are more similar cases.